vendor/engines/split_accounts/app/views/facility_accounts/account_fields/split_accounts/_splits.html.haml

```diff
@@ -0,0 +1,17 @@
+- available_subaccounts = Account.excluding_split_accounts.filter_global_account_types
```

Comment:
This should exclude expired and suspended accounts. I would almost consider creating a method on `SplitAccount.available_subaccounts` or something like that to wrap this up so it's not in the view.